.github/workflows/coverage.yml

```diff
@@ -25,19 +25,16 @@ jobs:
           - name: sev
             path: .
             host: [ self-hosted, linux, sev-snp ]
-            flags: --no-default-features --features=backend-sev
           - name: sgx
             path: .
             host: [ self-hosted, linux, sgx ]
-            flags: --no-default-features --features=backend-sgx
           - name: kvm
             path: .
             host: [ self-hosted, linux ]
-            flags: --no-default-features --features=backend-kvm
           - name: nil
             path: .
             host: [ self-hosted, linux ]
-            flags: --no-default-features --features=backend-nil --test integration --bin enarx
+            flags: "--test integration --bin enarx -- wasm::"
```

Comment:
Does this really work?

Reply:
yes... see https://github.com/enarx/enarx/runs/6350302648?check_suite_focus=true